src/main.cpp

```diff
@@ -651,11 +702,9 @@ int main(int argc, char **argv)
 		device.put("overrides.import.tls_pkey_path", "");
 		device.put("overrides.import.tls_clientcert_path", "");
 	}
-#ifdef AKLITE_TAGS
 	if (!options.pacman_tags.empty()) {
```

Comment:
I think this `if` is redundant since it's an exceptional situation and we should exit with error somewhere above. Either is the `_set_tag_options` as you did or in the new assert function`_assert_tags`.

Reply:
fixed